### PR TITLE
chore!: rename QGB to BlobStream

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/blobstream-contracts/issues/220

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: docs/inclusion-proofs.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# QGB Fraud Proofs
+# BlobStream Fraud Proofs
 
-## Celestiums Intro
+## BlobStream Intro
 
-A Celestium is a blockchain that uses Celestia for data availability but settles on any EVM chain. The QGB operates by having the Celestia validator set periodically sign over batched data commitments and validator set updates, which are relayed an EVM smart contract. The data commitments are stored in the EVM chain's state, and can be used to prove inclusion of any data historically posted to Celestia.
+A BlobStream rollup is a blockchain that uses Celestia for data availability but settles on any EVM chain. The BlobStream operates by having the Celestia validator set periodically sign over batched data commitments and validator set updates, which are relayed an EVM smart contract. The data commitments are stored in the EVM chain's state, and can be used to prove inclusion of any data historically posted to Celestia.
```
Comment (SweeXordious) [inline docs/inclusion-proofs.md:5]: [note to reviewers]
I changes `Celestium` to `BlobStream rollup`, not sure if that the right name or there is another name that's being decided upon

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-[![License: Apache-2.0](https://img.shields.io/github/license/celestiaorg/quantum-gravity-bridge.svg?style=flat-square)](https://github.com/celestiaorg/quantum-gravity-bridge/blob/master/LICENSE)
+[![GoDoc](https://img.shields.io/badge/godoc-reference-blue?style=flat-square&logo=go)](https://godoc.org/github.com/celestiaorg/blobstream-contracts)
+[![Go Report Card](https://goreportcard.com/badge/github.com/celestiaorg/blobstream-contracts?style=flat-square)](https://goreportcard.com/report/github.com/celestiaorg/blobstream-contracts)
+[![Version](https://img.shields.io/github/tag/celestiaorg/blobstream-contracts.svg?style=flat-square)](https://github.com/celestiaorg/blobstream-contracts/releases/latest)
+[![License: Apache-2.0](https://img.shields.io/github/license/celestiaorg/blobstream-contracts.svg?style=flat-square)](https://github.com/celestiaorg/blobstream-contracts/blob/master/LICENSE)
 
-The Quantum Gravity Bridge (QGB) is a Celestia -> EVM message relay.
+The Blobstream is a Celestia -> EVM message relay.
```
Comment (adlerjohn) [inline README.md:10]: Drop the "The," it's cleaner :)

(in other places too, when referring to Blobstream vs the Blobstream contract!)

```suggestion
Blobstream is a Celestia -> EVM message relay.
```

**Review by evan-forbes — APPROVED** [review]
this shouldn't break the application should it? maybe only the deploy command?

**Comment by SweeXordious** [comment]
> this shouldn't break the application should it? maybe only the deploy command?

Yes, only the deploy command but it's okey

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**